packages/framework/aqueduct/src/test/types/validateAqueductPrevious.ts

```diff
@@ -348,7 +348,6 @@ declare function get_current_InterfaceDeclaration_IDataObjectProps():
 declare function use_old_InterfaceDeclaration_IDataObjectProps(
     use: TypeOnly<old.IDataObjectProps>);
 use_old_InterfaceDeclaration_IDataObjectProps(
-    // @ts-expect-error compatibility expected to be broken
     get_current_InterfaceDeclaration_IDataObjectProps());
```

Comment:
@anthony-murphy This is a similar thing to what I see in the yarn branch. TS starts saying this isn't needed.